ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activityEmailAnswer.component.ts

```diff
@@ -0,0 +1,106 @@
+import { Component, EventEmitter, Input, OnChanges, OnDestroy, OnInit, Output, SimpleChanges } from '@angular/core';
+import { FormControl, Validators } from '@angular/forms';
+import * as _ from 'underscore';
+import { combineLatest, Observable, of, Subscription } from 'rxjs';
+import { ActivityEmailQuestionBlock } from '../../models/activity/activityEmailQuestionBlock';
+import { distinctUntilChanged, map, skip, startWith, tap } from 'rxjs/operators';
+import { NGXTranslateService } from '../../services/internationalization/ngxTranslate.service';
+
+@Component({
+  selector: 'ddp-activity-email-answer',
+  template: `
+      <ddp-question-prompt [block]="block"></ddp-question-prompt>
+      <mat-form-field class="example-input-field" [floatLabel]="block.label ? 'always' : null">
+          <mat-label *ngIf="block.label" [innerHTML]="block.label"></mat-label>
+          <input matInput
+                 type="email"
+                 [formControl]="formControls[0]"
+                 [minlength]="block.minLength"
+                 [maxlength]="block.maxLength"
+                 [placeholder]="placeholder || block.placeholder"
+                 [attr.data-ddp-test]="'answer:' + block.stableId">
+      </mat-form-field>
+      <mat-form-field class="example-input-field">
+          <input matInput
+                 type="email"
+                 [formControl]="formControls[1]"
+                 [minlength]="block.minLength"
+                 [maxlength]="block.maxLength"
+                 [placeholder]="confirmationPlaceholder | async"
+                 [attr.data-ddp-test]="'answer:' + block.stableId">
+      </mat-form-field>
+  `,
+  styles: []
+})
+export class ActivityEmailAnswerComponent implements OnInit, OnChanges, OnDestroy {
+  @Input() block: ActivityEmailQuestionBlock;
+  @Input() placeholder: string;
+  @Input() readonly: boolean;
+  @Output() valueChanged: EventEmitter<string | null> = new EventEmitter();
+  public formControls: FormControl[];
+  public confirmationPlaceholder: Observable<string | null>;
+  private subscription: Subscription;
+
+  constructor(private translate: NGXTranslateService) {
+  }
+
+  ngOnInit(): void {
+    this.initFormControls();
+    this.initPlaceholders();
+  }
+
+  private initFormControls() {
+    this.formControls = [0, 1].map(_ => new FormControl({
+      value: this.block.answer,
+      disabled: this.readonly
+    }, [Validators.email]));
```

Comment:
Reactive component includes email validator, so used reactive instead of ngModel. Also more flexibility.